cas-server-core/src/main/java/org/jasig/cas/CentralAuthenticationServiceImpl.java

```diff
@@ -216,7 +218,9 @@ public CentralAuthenticationServiceImpl(final TicketRegistry ticketRegistry,
             action="TICKET_GRANTING_TICKET_DESTROYED",
             actionResolverName="DESTROY_TICKET_GRANTING_TICKET_RESOLVER",
             resourceResolverName="DESTROY_TICKET_GRANTING_TICKET_RESOURCE_RESOLVER")
-    @Profiled(tag = "DESTROY_TICKET_GRANTING_TICKET", logFailuresSeparately = false)
+    @Timed(name = "DESTROY_TICKET_GRANTING_TICKET_TIMER")
+    @Metered(name="DESTROY_TICKET_GRANTING_TICKET_METER")

```

Comment:
A meter measures the rate at which a set of events occur. Meters measure the rate of the events in a few different ways. The mean rate is the average rate of events. It’s generally useful for trivia, but as it represents the total rate for your application’s entire lifetime (e.g., the total number of requests handled, divided by the number of seconds the process has been running), it doesn’t offer a sense of recency. Luckily, meters also record three different exponentially-weighted moving average rates: the 1-, 5-, and 15-minute moving averages.

https://dropwizard.github.io/metrics/3.1.0/manual/core/#meters